Taskfile.yml

```diff
@@ -300,6 +300,12 @@ tasks:
       - task: install-playground-npm-dependencies
       - npm start
 
+  run-pg:
```

Comment:
This is OK! I would have voted for something clearer — not obvious how `pg` is different from `playground` (though it is elegant in a way! Smaller and less descriptive, just like running vs building!!)